<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggerWidget</class>
 <widget class="QWidget" name="LoggerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="debugCheckBox">
       <property name="text">
        <string>DEBUG</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="errorCheckBox">
       <property name="text">
        <string>ERROR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="warningCheckBox">
       <property name="text">
        <string>WARNING</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="infoCheckBox">
       <property name="text">
        <string>INFO</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="msgConsole">
     <property name="styleSheet">
      <string notr="true">background-color: black</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
